filebeat/docs/fields.asciidoc

```diff
@@ -28240,6 +28240,15 @@ type: keyword
 Palo Alto Networks name for the threat.
 
 
+type: keyword
+
+--
+
+*`panw.panos.action`*::
++
+--
+Action taken for the session.
+
```

Comment:
Should this change be in this PR? I think I saw a separate PR recently to fix this: https://github.com/elastic/beats/pull/17947/files.

Reply:
Great. I did not know about the fix. Thank you.